web/src/components/icon.jsx

```diff
@@ -19,12 +13,12 @@ const path = window.isCordova ? resolvePath('images/material-icon-sprite.svg') :
  *   </symbol>
  */
 
-const Icon = ({ iconName }) => {
+const Icon = memo(({ iconName, size = 24, color }) => {
   return (
-    <IconContainer>
+    <svg height={size} width={size} fill={color}>
```

Comment:
Nice, and overriding with css (e.g. on `hover`) will still have higher specificity, right?

Reply:
Ah great! 🥳